integ-test/src/test/java/org/opensearch/sql/legacy/OpenSearchSQLRestTestCase.java

```diff
@@ -105,10 +105,10 @@ protected RestClient buildClient(Settings settings, HttpHost[] hosts) throws IOE
   }
 
   // Modified from initClient in OpenSearchRestTestCase
-  public void initRemoteClient() throws IOException {
+  public void initRemoteClient(String clusterName) throws IOException {
```

Comment:
Does this not reduce potential future reuse of this function?

Reply:
The cluster name is defined in the gradle task.  Sean hardcoded the name in a static variable, so this only used to work with the remoteCluster, and now it works with whatever the gradle task has named the cluster. 
So... no.  I think this is more reusable?